src/cmd_line/subparsers/file.ts

```diff
@@ -9,7 +9,7 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     }
 
     var scanner = new Scanner(args);
-    let name = scanner.nextWord();
+    var name = scanner.nextWord();
```

Comment:
My thoughts about variable types:

* Use `const` as much as possible
* When `const` is impossible (the variable gets changed), use `let`

Even though `let` and `var` are equivalent in this case, `let` is generally superior to `var` and there is no disadvantage to using `let; therefore we should always use `let`.

I'll whip up a style guide soon. :smile: